### ISSUE TITLE
Creating new channel or page in laravel integrated forum

### ISSUE THREAD

**Issue by keithk** [waterholeforum/core#55]
### Bug description

500 from `/cp/structure/pages` when attempting to create a page or channel (but not link or heading!). It appears that it's trying to connect to the laravel DB, instead of the waterhole identified DB.

### Steps to reproduce

1. Install a Waterhole into an existing site with a separate DB for each
2. Navigate to the CP with the admin user
3. Try to create a new channel or page and no matter what you enter you'll get a 500 from `/cp/structure/pages`

### Logs

```shell
┌ 2024-01-07 01:46:00 Illuminate\Database\QueryException  vendor/laravel/framework/src/Illuminate/Database/Connection.php:822
│ SQLSTATE[42S02]: Base table or view not found: 1146 Table 'SITE_DB.pages' doesn't exist (Connection: mysql, SQL: select count(*) as aggregate from `pages` where `slug` = can-i-create-a-page)
└───────────────────────────────────────────────────────────────── POST: /cp/structure/pages • Auth ID: 2
```

### Environment

```yaml
php artisan about --only=environment

  Environment ..........................................................................................
  Application Name ............................................................................. 🤖 Kumos 
  Laravel Version .............................................................................. 10.39.0
  PHP Version ................................................................................... 8.2.14
  Composer Version ............................................................................... 2.5.5
  Environment .................................................................................... local
  Debug Mode ................................................................................... ENABLED
  URL ....................................................................................... kumos.test
  Maintenance Mode ................................................................................. OFF
  Waterhole Version .............................................................................. 0.4.2
```

**Comment by kernwdid** [comment]
Thank you for opening this ticket 👍 I run into the same issue.

**Comment by tobyzerner** [comment]
@keithk @kernwdid I've just released 0.4.3 with a fix for this issue!
